### PR TITLE
core: text cleanup

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

File: spec/core/state.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-Note that the `State` object itself is an implementation detail, since it is never
+The `State` object itself is an implementation detail, since it is never
 included in a block or gossiped over the network, and we never compute
-its hash. Thus we do not include here details of how the `State` object is
-persisted or queried. That said, the types it contains are part of the specification, since
-their Merkle roots are included in blocks and their values are used in
+its hash. The persisence or query interface of of the `State` object
+is an implementation detail and not included in the specification.
+However, the types in the `State` object are part of the specification, since
+the Merkle roots of the se objects are included in blocks and values are used during
```
Comment (creachadair) [inline spec/core/state.md:13]: > the se objects

Was this a truncation of "state"?

Comment (tychoish) [reply spec/core/state.md:13]: yup

File: spec/core/state.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+transactions are never included in blocks, but their Merkle roots are:
+the state keeps track of them.
 
-Note that the `State` object itself is an implementation detail, since it is never
+The `State` object itself is an implementation detail, since it is never
 included in a block or gossiped over the network, and we never compute
-its hash. Thus we do not include here details of how the `State` object is
-persisted or queried. That said, the types it contains are part of the specification, since
-their Merkle roots are included in blocks and their values are used in
+its hash. The persisence or query interface of of the `State` object
```
Comment (josef-widder) [inline spec/core/state.md:10]:
```suggestion
its hash. The persistence or query interface of the `State` object
```

File: spec/core/state.md [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-Note that the `State` object itself is an implementation detail, since it is never
+The `State` object itself is an implementation detail, since it is never
 included in a block or gossiped over the network, and we never compute
-its hash. Thus we do not include here details of how the `State` object is
-persisted or queried. That said, the types it contains are part of the specification, since
-their Merkle roots are included in blocks and their values are used in
+its hash. The persistence or query interface of the `State` object
+is an implementation detail and not included in the specification.
+However, the types in the `State` object are part of the specification, since
+the Merkle roots of the `State`objects are included in blocks and values are used during
```
Comment (creachadair) [inline spec/core/state.md:13]:
```suggestion
the Merkle roots of the `State` objects are included in blocks and values are used during
```

**Review by creachadair — APPROVED** [review]

**Pull request merged**